scripts/release/publish-commands/print-follow-up-instructions.js

```diff
@@ -50,7 +46,13 @@ const run = async ({cwd, packages, tags}) => {
         const packageName = packages[i];
         console.log(theme.path`• packages/%s/package.json`, packageName);
       }
-      console.log(theme.path`• packages/shared/ReactVersion.js`);
+      const status = await execRead(
+        'git diff packages/shared/ReactVersion.js',
+        {cwd}
+      );
+      if (status) {
```

Comment:
It's possible this version wasn't actually updated (depending on what we skipped) so I added a check to avoid printing a confusing message.